core/src/test/scala/restui/models/EventSpec.scala

```diff
@@ -0,0 +1,26 @@
+package restui.models
+
+import io.circe.syntax._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.prop.TableDrivenPropertyChecks
+import restui.models.Event._
+
+class EventSpec extends AnyFlatSpec with Matchers with TableDrivenPropertyChecks {
+  private val properties = Table(
+    ("event", "json"),
+    (Event.ServiceUp("id", "test", Map("key" -> "value")), """{"event":"serviceUp","id":"id","name":"test","metadata":{"key":"value"}}"""),
+    (Event.ServiceDown("id"), """{"event":"serviceDown","id":"id"}"""),
+    (List(Event.ServiceUp("id", "test", Map("key" -> "value"))),
```

Comment:
I would say list doesnt have to belog in this set  and maybe create another table with list of service up and service down